src/events/DistanceEvents.ts

```diff
@@ -26,7 +33,7 @@ export interface IPositionResolvedEventArgs {
   /**
    * The position that was resolved.
    */
-  position: IPosition;
+  position: IGeolocationPosition;
```

Comment:
I think it would be a good idea to also rename the interface in the PopupUtils class to something like IPopupPosition.